en/deploy-on-aws-eks.md

```diff
@@ -529,7 +529,7 @@ For more information about the storage class configuration and EBS volume types,
 
 ## Use local storage
 
-Some AWS instance types provide additional [NVMe SSD local store volumes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ssd-instance-store.html). You can choose such instances for the TiKV node pool to achieve higher IOPS and lower latency.
+Use AWS EBS as a primary production configuration. To simulate bare-metal performance, some AWS instance types provide additional [NVMe SSD local store volumes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ssd-instance-store.html). You can choose such instances for the TiKV node pool to achieve higher IOPS and lower latency.
```

Comment:
```suggestion
Use AWS EBS as a primary production configuration. To simulate bare metal performance, some AWS instance types provide additional [NVMe SSD local store volumes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ssd-instance-store.html). You can choose such instances for the TiKV node pool to achieve higher IOPS and lower latency.
```

We recommend `bare metal` as in [Google's word list](https://developers.google.cn/style/word-list#letter-b).